src/api/app/views/layouts/webui2/_personal_navigation.html.haml

```diff
@@ -0,0 +1,61 @@
+- if User.current && !User.current.is_nobody?
+  %li.nav-item
+    = link_to(home_path, id: 'link-to-user-home', class: 'nav-link') do
+      = User.current.login
+  %li.nav-item
+    - tasks = User.current.tasks
+    = link_to(user_tasks_path, class: 'nav-link') do
+      - if tasks > 0
```

Comment:
[`Style/NumericPredicate` Rubocop cop](https://rubocop.readthedocs.io/en/latest/cops_style/#stylenumericpredicate) doesn't like this... :sob: it should be `tasks.positive?`